pkgs/by-name/ba/banana-accounting/package.nix

```diff
@@ -38,25 +40,24 @@ stdenv.mkDerivation {
     libgcrypt
   ];
 
-  nativeBuildInputs = [
-    autoPatchelfHook
-    wrapGAppsHook3
-  ];
+  dontConfigure = true;
+
+  dontBuild = true;
 
   installPhase = ''
     runHook preInstall
-    mkdir -p $out/bin
-    mv ./* $out
-    ln -s $out/usr/bin/bananaplus $out/bin/
+
+    cp -r usr $out
```

Comment:
This change seems to spam $out/share with paths that may collide in a fhs env, there is also the qml file in $out/bin that ideally would not be there, is it possible to for example put everything into $out/opt and link it where it should be?